src/components/EnvelopeSkeleton.vue

```diff
@@ -595,6 +599,10 @@ export default {
 		.list-item__actions {
 			align-self: center;
 			margin-top: 0;
+
+			&__filler {
+				margin-bottom: -10.25px;
```

Comment:
nitpick: Is this a stable solution in a sense that it won't break with any small pixel shifts of future changes of this component?

![](https://images.squarespace-cdn.com/content/5528c6abe4b0c75f5b823f60/1429309416071-BOPD45JFJI5IL88VPDU9/Skeptical-Hippo.jpg?content-type=image%2Fjpeg)

Reply:
Hey @ChristophWurst! Yeah, I guess that it might change 😅, I changed it now to get the number through a dynamic calculation, the formula is a little weird because of the complexity of the formatting here (which is why I didn't think of using it initially, but I left a comment explaining it, so people don't get too confused in the future), but it does make sense.

Reply:
Whohoo